src/components/storage/CockpitStorageIntegration.jsx

```diff
@@ -119,8 +119,16 @@ export const CockpitStorageIntegration = ({
     showStorage,
 }) => {
     const [showDialog, setShowDialog] = useState(false);
+    const [showConfirmation, setShowConfirmation] = useState(false);
+    const [confirmed, setConfirmed] = useState(false);
```

Comment:
Using is as a prefix for boolean state variables is a common convention in React and JavaScript coding practices.

For consistency purpose please change this to [isConfirmed, set..]